src/Components/FAQ.js

```diff
@@ -62,118 +62,225 @@ const FAQ = props => {
         <Gradient>
           <SingleItem>
             <Collapsible
-              trigger="Who the hell is Ray Dalio?"
+              trigger="What is Pie?"
```

Comment:
Would suggest to change it to "What is the Pie Network". This avoids confusion with the token which ticker will be PIE